src/blueprint.js

```diff
@@ -47,6 +49,172 @@ function generateSnapshot () {
     return global
   }
 
+  // Globally visible function and constructor names that are available in an Electron renderer window, but not visible
+  // during snapshot creation.
+  // See test/samples/list-globals.js for the generation code.
+  const globalFunctionNames = [
```

Comment:
In the future, should we generate this dynamically for the user's target Electron version in case the list changes over time?

Reply:
Oh yeah, that would be awesome ✨ 

I'm not sure exactly how we'd do that though - we need to get the set of global functions available from `mksnapshot` (`Array`, `Object`, and so forth) and the set of global functions available in a renderer process for the target Electron version, then generate the difference. The challenge is cleanly getting access to both of those environments at electron-link time, executing code there, and extricating a result.

For the moment, my instinct is that (a) this set doesn't change that often and (b) having the wrong one isn't _that_ bad?